meson.build

```diff
@@ -60,6 +52,31 @@ sys_bsd = bsd.contains(host_machine.system())
 sys_windows = windows.contains(host_machine.system())
 sys_osx = osx.contains(host_machine.system())
 
+if sys_windows
+  nuget_download = get_option('nuget-download')
+  if not nuget_download
+    message('needs -Dnuget-download=true to download deps')
+  endif
+  if cc.has_header_symbol('stdio.h', '_MSC_VER') and nuget_download
```

Comment:
This could be:
```suggestion
  elif cc.has_header_symbol('stdio.h', '_MSC_VER') 
```

Reply:
I think not, nuget should only be downloaded (line below) if `nuget_download` is set to true.

Reply:
Yeah, how it is now it doesn't make sense.